x-pack/solutions/security/plugins/security_solution/public/entity_analytics/pages/entity_analytics_dashboard.tsx

```diff
@@ -58,6 +58,10 @@ const EntityAnalyticsComponent = () => {
   const showEmptyPrompt = !indicesExist && !skipEmptyPrompt;
   const entityTypes = useEntityAnalyticsTypes();
 
+  if (newDataViewPickerEnabled && status === 'pristine') {
+    return null;
```

Comment:
Tested with some larger dataViews, and we should show some loader here. We can sync with design, but thinking creating a shared component like this would work: https://eui.elastic.co/docs/components/display/empty-prompt/#loading-and-error-prompts


```
const LOADING_TITLE = (pageName: string) => i18n.translate('xpack.securitySolution.page.loader',
    {
      defaultMessage: 'Loading {pageName}',
      values: { pageName },
    });
    
const SecurityPageLoader = (pageName: string) => (
       <EuiEmptyPrompt
          icon={<EuiLoadingLogo logo="logoSecurity" size="xl" />}
          title={<h2>{LOADING_TITLE(pageName)}</h2>}
        />
}
```

or something like this...